src/java/org/padaiyal/wiki/performance/CpuHoggerScenario.java

```diff
@@ -0,0 +1,41 @@
+public class CpuHoggerScenario {
```

Comment:
Add documentation comments for class and methods.

Reply:
In the class documentation comment, explain as to what the sample code is doing in detail.